src/components/Renderer/index.tsx

```diff
@@ -25,14 +25,13 @@ export const getValue = (
 	schema?: JSONSchema,
 	uiSchema?: UiSchema,
 ) => {
-	const calculatedValue = get(uiSchema, 'ui:value', value);
+	const calculatedValue = uiSchema?.['ui:value'] ?? value;
```

Comment:
These `get` changes were primarily to clean up/simplify the code so it was easier to see what was going on once it has all been transpiled/etc and it does also improve the ability for typescript to apply type-checking